debugprofiler.md

```diff
@@ -0,0 +1,52 @@
+Forge Debug Profiler oraz jego interpretacja
+===============================================
+
+Spis treści
+-----------
+* [Podstawy](#podstawy)
+* [Jak używać debugera](#Jak_używać_debugera)
+* [Czytanie Profilera](#czytanie)
+
+## Podstawy
+Minecraft dostarcza nam `Debug Profiler` dzięki któremu można sprawdzić obciążający kod.
+Szczególnie chodzi tutaj o rzeczy pokroju `TickEvent` lub `Ticking TileEntities`, 
+przydaje się to w szcególności modderom, jak także właścicielom serwerów, którzy chcą sprawdzić przyczynę lagów.
+
+<a name="Jak_używać_debugera"><h2>Jak używać profilera</h2></a>
+
+Profiler jest banalnie prosty w obsłudze:
+- `/debug start` włącza nam Profiler (z konsoli, lub bezpośrednio w grze*)
```

Comment:
Warto dopisać, że w konsoli komendy wpisuje się bez "/". Dla wielu użytkowników jest to bardzo poważny problem.